lib/simplecov.rb

```diff
@@ -189,6 +189,42 @@ def exit_status_from_exception
         SimpleCov::ExitCodes::EXCEPTION
       end
     end
+
+    #
+    # Usage:
+    #   exit_status = SimpleCov.process_result(SimpleCov.result, exit_status)
+    #
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/PerceivedComplexity
+    # rubocop:disable Metrics/AbcSize
+    def process_result(result, exit_status)
+      covered_percent = result.covered_percent.round(2)
+      covered_percentages = result.covered_percentages.map { |p| p.round(2) }
```

Comment:
I put these before the guard clause only because that's how they are in the original.  Unless they have some important side effect I am unaware of, they should be put below it.

Reply:
it shouldn't have... wanna move it to the appropriate place? If tests fail, we can still go back otherwise I'd claim we are good.

Personally I'd also prefer a renaming to `percent` - `p` always makes me think of the `p` print method